src/Components/Screens/Gameplay/Chat/chat.scss

```diff
@@ -29,25 +29,27 @@
     position: relative;
     .chat-options {
       position: absolute;
-      bottom: -100px;
-      left: 10px;
+      bottom: -27px;
       display: flex;
-      flex-direction: column;
-      border-radius: 3px;
+      border-radius: 2px;
+      padding: 5px 3px;
+      background-image: url(http://localhost:3000/static/media/main_pj_info.9972debe9c5fc5a449ec.png);
```

Comment:
done!
changes:
-replaced '.chat-options''.chat-opt' bg images with images from assets folder.
-'.chat-opt:hover' using a bg image instead of opacity.
-modified "selection-icon" image. Created a new icon 'ico_active_chat' and stored it inside of "assets/icons/gameplay".